src/core/Projects/MsBuildProject.ts

```diff
@@ -1,3 +1,4 @@
+import { Project as NugetProject } from 'nuget-deps-tree/dist/src/Structure/OutputStructure';
```

Comment:
I've created an issue to get these types exported: https://github.com/jfrog/nuget-deps-tree/issues/24

Reply:
I am going to modify the `nuget-deps-tree` dependency and write my own code to read the `project.assets.json` file. So this is not a problem :)